### PR TITLE
DRILL-7614: Add some helpful tips to Testing.md

### PR DESCRIPTION
**Pull request by dobesv**
Just a little docs change.

### CODE REVIEW

File: docs/dev/Testing.md [hunk 1]
```diff
@@ -172,6 +172,59 @@ class DrillBuf ...
...
+    
+    DRILL_HOST_NAME=localhost:8047
+    DRILL_LOG_DIR=/opt/drill/log
+
+  4. Edit `/opt/drill/conf/drill-override.conf` to set some options:
+  
+    drill.exec: {
+      zk.connect: "localhost:2181",
+      allow_loopback_address_binding: true,
+    }
```
Comment (paul-rogers) [inline docs/dev/Testing.md]: Another FYI: Another option is to create a `ClusterTest` based test that does this. That way, all the config is in a code file and you just tell it to run. Just another option.

File: docs/dev/Testing.md [hunk 2]
```diff
@@ -172,6 +172,59 @@ class DrillBuf ...
...
+    -Xmx4G
+    -XX:MaxDirectMemorySize=8G
+    -XX:ReservedCodeCacheSize=1G
+    -Ddrill.exec.enable-epoll=false
+    -XX:MaxPermSize=512M
+    -XX:+CMSClassUnloadingEnabled
+    -XX:+UseG1GC
+    -Dlog.path=/opt/drill/log/drillbit.log
+    -Dlog.query.path=/opt/drill/log/drillbit_queries.json
+    -Xbootclasspath/a:/opt/drill/conf
```
Comment (paul-rogers) [inline docs/dev/Testing.md]: This is only needed to simulate a "real" drillbit. In that case, I've found it easier to just add an `DRILLBIT_JAVA_OPTS` variable, in the environment or in `drill-env.sh` to enable remote debugging. Then, just connect to the running Drillbit from your IDE and you are assured to be using all the right config.

What is describe here works, just is a bit fiddly.

You can also see that config with `drillbit.sh debug`.

**Comment by arina-ielchiieva** [comment]
@dobesv we don't need to create Jira if we are doing updates to the Drill site but for docs that reside in Drill project, Jira is required. I have already created Jira for you (https://issues.apache.org/jira/browse/DRILL-7614). Could you please update the commit message to include Jira number? Thanks!

**Comment by cgivre** [comment]
Just a suggestion, but could we put this in the Drill website as well in the developer tab?

File: docs/dev/Testing.md (hunk 1 above)
Comment (dobesv) [reply docs/dev/Testing.md]: Hmm if you have changes to suggest I guess you can open a PR against this branch?

**Comment by dobesv** [comment]
I agree this might be nice to have in the website, to be honest I only found this when I was thinking I wanted to document it, I hadn't seen that developer docs folder before.

**Comment by cgivre** [comment]
@dobesv That wasn't for this PR... I'm not really sure that we have a process for website updates. @bbevens Can you assist?

**Comment by arina-ielchiieva** [comment]
@cgivre I think Bridget is no longer actively engaged in Drill project. 
Well, currently Testing.md information is available only inside Drill project, of course it can be copied to Drill site, there is an extensive tutorial on how to generate web site updates (https://github.com/apache/drill/blob/gh-pages/README.md), first of course PR should be opened to the site branch. Plus @vvysotskyi is working on the automated job which will update Drill site once site branch is updated ( https://issues.apache.org/jira/browse/DRILL-7571).

**Pull request closed without merging**